cord19-explorer/src/main/webapp/WEB-INF/views/paper.html

```diff
@@ -66,7 +66,11 @@ <h3 th:text="*{section.title != ''} ? *{section.title}: 'No Section title'"></h3
 	      $(function() {
 	      	[# th:each="match : ${para.locationMatches}" ]
 	      	[# th:if="${match != null && match.selectedArticle != null}" ]
-	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false})	
+	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false})
+	     	var title = [[${match.selectedArticle.title}]]
+	      	$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false}).attr('title', title);
+	        //$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false}).wrap( "<a href='https://en.wikipedia.org/wiki/"+title+"' target=_blank></a>" );
+	        $( "mark" ).wrap( "<a href='https://en.wikipedia.org/wiki/"+title+"' target=_blank></a>" );
```

Comment:
have you tried adding a span with a class that contains for example the index of the loop and use javascript to generate the link on click. something like this:
```
$("#[(${#ids.next('para')})]").mark("[(${match.locationName})]", { "separateWordSearch": false, "className": "wiki-[(${stats.index})]"})
$("mark.wiki-[(${stats.index})]).click(...)
```
does this make sense? this is about iterations in thymeleaf: https://www.baeldung.com/thymeleaf-iteration, and this is the mark.js documentation: https://markjs.io/